application/Profile/LocalProfile.py

```
@@ -20,10 +20,12 @@
 import base64
 import copy
 import json
+import logging
 import uuid
 
 from application.paths import *
 from .Profile import Profile
+from typing import Any, Dict, Tuple
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)